cosmwasm/contracts/token-bridge/src/contract.rs

```diff
@@ -872,7 +872,10 @@ fn handle_complete_transfer_token(
     let (not_supported_amount, mut amount) = transfer_info.amount;
     let (not_supported_fee, mut fee) = transfer_info.fee;
 
-    amount = amount.checked_sub(fee).unwrap();
+    amount = match amount.checked_sub(fee) {
+        None => return Result::Err(cosmwasm_std::StdError::generic_err("Insufficient funds")),
+        Some(value) => value
+    };
```

Comment:
style nit: I think `ok_or_else` is the more idiomatic way to do this:

```suggestion
    amount = amount
        .checked_sub(fee)
        .ok_or_else(|| StdError::generic_err("Insufficient funds"))?;
```

same for the other places where we do this.